runner_files/crack.rb

```diff
@@ -0,0 +1,11 @@
+require './lib/enigma'
+
+enigma = Enigma.new
+message_reader = File.open(ARGV[0], "r")
+message_to_decrypt = message_reader.read.chomp
```

Comment:
chomp is super important here, took some prying to figure out why crack wasn't working